packages/scripts/src/upgrade-tests/voting2/0_Deploy.ts

```diff
@@ -76,18 +76,18 @@ async function main() {
 
   console.log("DVM2.0 Parameters:");
   console.table({
-    emergencyQuorum: hre.ethers.utils.formatUnits(emergencyQuorum, "ether"),
+    emergencyQuorum: hre.ethers.utils.parseUnits(emergencyQuorum, "ether"),
```

Comment:
I don't think this should be changed here as `formatUnits` did produce human readable output, while `parseUnits` would add additional 18 decimals to the value that is already in wei